src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java

```diff
@@ -42,38 +41,16 @@ public void systemLoadsManageConditions() {
 		assertNotNull(getElement(addNewCondition));
 	}
 
-	@And("User clicks on Return")
```

Comment:
Are you sure we dont need these step definitions

Reply:
> Are you sure we dont need these step definitions

None of the scenarios requires the system to first go back to the patient dashboard. When that background runs, it comes to the Manage Conditions Page and that's where all other scenarios pickup from